doc/configuration.rst

```diff
@@ -143,6 +143,56 @@ You can specify the following options in module configuration.
         position = 'center'
     }
 
+.. note::
+    New in version 3.16
+
+You can specify the options in module or py3status configuration section.
+
+The following options will work on ``i3``.
+
+``align``: Specify how modules should be aligned when the ``min_width``
+``background``: Specify a background color for modules.
+is not reached. Either ``left`` (default), ``center``, or ``right``.
```

Comment:
mix up here

Reply:
line 155 is the sentence continuation from line 153

Reply:
I see. That kind of mix up. :laughing: